CHANGELOG.next.asciidoc

```diff
@@ -112,24 +71,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Affecting all Beats*
 
-- Add a friendly log message when a request to docker has exceeded the deadline. {pull}15336[15336]
 - Decouple Debug logging from fail_on_error logic for rename, copy, truncate processors {pull}12451[12451]
 - Allow a beat to ship monitoring data directly to an Elasticsearch monitoring cluster. {pull}9260[9260]
 - Updated go-seccomp-bpf library to v1.1.0 which updates syscall lists for Linux v5.0. {pull}11394[11394]
 - add_host_metadata is no GA. {pull}13148[13148]
 - Add `providers` setting to `add_cloud_metadata` processor. {pull}13812[13812]
-- GA the `script` processor. {pull}14325[14325]
-- Add `fingerprint` processor. {issue}11173[11173] {pull}14205[14205]
-- Add support for API keys in Elasticsearch outputs. {pull}14324[14324]
-- Ensure that init containers are no longer tailed after they stop {pull}14394[14394]
```

Comment:
@mikemadden42 I can't find this line in the `CHANGELOG.asciidoc` file. Shouldn't it be there?

Reply:
Never mind, looks like this was released in 7.5.0.